sources/packages/web/src/components/generic/Banner.models.ts

```diff
@@ -0,0 +1,6 @@
+export enum BannerTypes {
```

Comment:
Do we need to consume this outside the component?

Reply:
Yes. we use this enum to set the attribute while using the component.